### PR TITLE
made first solution

### PR DESCRIPTION
**Pull request by Sashk52**

### CODE REVIEW

File: src/main/java/task/second/ExecutorServiceSolution.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        }
+        int totalSum = 0;
+        List<Future<Integer>> futures = null;
+        try {
+            futures = executorService.invokeAll(callableTasks);
+            for (Future<Integer> future : futures) {
+                totalSum += future.get();
+            }
+            return totalSum;
+        } catch (InterruptedException e) {
```
Comment (IwalkInSilence) [inline src/main/java/task/second/ExecutorServiceSolution.java:37]: You can catch 2 exceptions in one block ` catch (InterruptedException | ExecutionException e) {`

**Review by IwalkInSilence — APPROVED** [review]

File: src/main/java/task/second/ExecutorServiceSolution.java (hunk 1 above)
Comment (Sashk52) [reply src/main/java/task/second/ExecutorServiceSolution.java:37]: I know, but i separate this to get more informative messages depends on type of exception.

File: src/main/java/task/second/CallableImpl.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package task.second;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class CallableImpl implements Callable<Integer> {
```
Comment (RomanDubovskyi) [inline src/main/java/task/second/CallableImpl.java]: let's name class according to its' purpose

File: src/main/java/task/second/ExecutorServiceSolution.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            throw new RuntimeException("Can't invoke all callableTasks", e);
+        } catch (ExecutionException e) {
+            throw new RuntimeException("Can't get all callableTasks", e);
+        } finally {
+            executorService.shutdown();
+        }
```
Comment (RomanDubovskyi) [inline src/main/java/task/second/ExecutorServiceSolution.java]: we don't need to call .shutdown() in finally. it's not like resources with files or db

File: src/main/java/task/second/ExecutorServiceSolutionTest.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
+package task.second;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ExecutorServiceSolutionTest {
```
Comment (RomanDubovskyi) [inline src/main/java/task/second/ExecutorServiceSolutionTest.java]: this class should it package with test not in main

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

File: src/main/java/task/second/CallableImpl.java (hunk 2 above)
Comment (Sashk52) [reply src/main/java/task/second/CallableImpl.java]: done

File: src/main/java/task/second/ExecutorServiceSolution.java (hunk 3 above)
Comment (Sashk52) [reply src/main/java/task/second/ExecutorServiceSolution.java]: done

File: src/main/java/task/second/ExecutorServiceSolutionTest.java (hunk 4 above)
Comment (Sashk52) [reply src/main/java/task/second/ExecutorServiceSolutionTest.java]: done

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**